__tests__/data/fb_custom_audience_input.json

```diff
@@ -472,5 +472,139 @@
+                "userSchema": [
+                    "EMAIL",
+                    "DOBM",
+                    "DOBD",
+                    "DOBY",
+                    "PHONE",
+                    "GEN",
+                    "FI",
+                    "MADID",
+                    "ZIP",
+                    "ST",
+                    "COUNTRY"
+                ],
+                "isHashRequired": true,
+                "disableFormat": false,
+                "skipVerify": true,
+                "audienceId": "aud1",
+                "isRaw": true,
+                "type": "UNKNOWN",
+                "subType": "ANYTHING",
+                "maxUserCount": "50"
+            },
+            "Enabled": true,
+            "Transformations": [],
+            "IsProcessorEnabled": true
+        },
+        "libraries": [],
+        "request": {
+            "query": {}
+        }
+    },
+    {
+        "message": {
+            "userId": "user 1",
+            "anonymousId": "anon-id-new",
+            "event": "event1",
+            "type": "audiencelist",
+            "properties": {
+                "listData": {
+                    "add": [
+                        {
+                            "EMAIL": "shrouti@abc.com",
+                            "DOBM": "2",
+                            "DOBD": "13",
+                            "DOBY": "2013",
+                            "PHONE": null,
+                            "GEN": "f",
+                            "FI": "Ms.",
+                            "MADID": "ABC",
+                            "ZIP": "ZIP ",
+                            "ST": "123abc ",
+                            "COUNTRY": "IN"
+                        }
+                    ]
+                }
+            },
+            "context": {
+                "ip": "14.5.67.21",
+                "library": {
+                    "name": "http"
+                }
+            },
+            "timestamp": "2020-02-02T00:23:09.544Z"
+        },
+        "destination": {
+            "Config": {
+                "accessToken": "ABC",
+                "userSchema": [
+                    "EMAIL",
+                    "DOBM",
+                    "DOBD",
+                    "DOBY",
+                    "PHONE",
+                    "GEN",
+                    "FI",
+                    "MADID",
+                    "ZIP",
+                    "ST",
+                    "COUNTRY"
+                ],
+                "isHashRequired": true,
+                "disableFormat": false,
+                "skipVerify": false,
+                "audienceId": "aud1",
+                "isRaw": true,
+                "type": "UNKNOWN",
+                "subType": "ANYTHING",
+                "maxUserCount": "50"
+            },
+            "Enabled": true,
+            "Transformations": [],
+            "IsProcessorEnabled": true
+        },
+        "libraries": [],
+        "request": {
+            "query": {}
+        }
     }
```

Comment:
Format this file with prettier like other files.

Reply:
This is done, you can resolve this conversation.